<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>998</width>
    <height>635</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>998</width>
    <height>635</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>998</width>
    <height>635</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>441</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiBold</family>
      <pointsize>32</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color : white;</string>
    </property>
    <property name="text">
     <string>Contact Manager</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-30</y>
      <width>1011</width>
      <height>681</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image.qrc">:/newPrefix/image/main.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="lstContacts">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>431</width>
      <height>491</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Atkinson Hyperlegible Next</family>
      <pointsize>11</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(8, 28, 75,0.4);
border: 3px solid white;
border-radius: 10px;
color:white;
font: 75 11pt &quot;Atkinson Hyperlegible Next&quot;;
padding-left:9px;
padding-top:9px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtName">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>110</y>
      <width>451</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Atkinson Hyperlegible Next</family>
      <pointsize>11</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(8, 28, 75,0.4);
border: 3px solid white;
border-radius: 10px;
color:white;
font: 75 11pt &quot;Atkinson Hyperlegible Next&quot;;
padding-left:9px;
</string>
    </property>
    <property name="placeholderText">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPhone">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>180</y>
      <width>451</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Atkinson Hyperlegible Next</family>
      <pointsize>11</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(8, 28, 75,0.4);
border: 3px solid white;
border-radius: 10px;
color:white;
font: 75 11pt &quot;Atkinson Hyperlegible Next&quot;;
padding-left:9px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Phone Number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtEmail">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>250</y>
      <width>451</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Atkinson Hyperlegible Next</family>
      <pointsize>11</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(8, 28, 75,0.4);
border: 3px solid white;
border-radius: 10px;
color:white;
font: 75 11pt &quot;Atkinson Hyperlegible Next&quot;;
padding-left:9px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Email </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtAddress">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>320</y>
      <width>451</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Atkinson Hyperlegible Next</family>
      <pointsize>11</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(8, 28, 75,0.4);
border: 3px solid white;
border-radius: 10px;
color:white;
font: 75 11pt &quot;Atkinson Hyperlegible Next&quot;;
padding-left:9px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Address</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEditContact">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>390</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Atkinson Hyperlegible Next</family>
      <pointsize>12</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(8, 28, 75,0.4);
border: 3px solid white;
border-radius: 10px;
color:white;
font: 75 12pt &quot;Atkinson Hyperlegible Next&quot;;
</string>
    </property>
    <property name="text">
     <string>Edit Contact</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDeleteContact">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>390</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(8, 28, 75,0.4);
border: 3px solid white;
border-radius: 10px;
color:white;
font: 75 12pt &quot;Atkinson Hyperlegible Next&quot;;
</string>
    </property>
    <property name="text">
     <string>Delete Contact</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLoadContacts">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>460</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Atkinson Hyperlegible Next</family>
      <pointsize>12</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(8, 28, 75,0.4);
border: 3px solid white;
border-radius: 10px;
color:white;
font: 75 12pt &quot;Atkinson Hyperlegible Next&quot;;
</string>
    </property>
    <property name="text">
     <string>Load Contacts</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSaveContacts">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>460</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Atkinson Hyperlegible Next</family>
      <pointsize>12</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(8, 28, 75,0.4);
border: 3px solid white;
border-radius: 10px;
color:white;
font: 75 12pt &quot;Atkinson Hyperlegible Next&quot;;
</string>
    </property>
    <property name="text">
     <string>Save Contacts</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddContact">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>530</y>
      <width>451</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Atkinson Hyperlegible Next</family>
      <pointsize>12</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(8, 28, 75,0.4);
border: 3px solid white;
border-radius: 10px;
color:white;
font: 75 12pt &quot;Atkinson Hyperlegible Next&quot;;
</string>
    </property>
    <property name="text">
     <string>Add Contact</string>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>label</zorder>
   <zorder>lstContacts</zorder>
   <zorder>txtName</zorder>
   <zorder>txtPhone</zorder>
   <zorder>txtEmail</zorder>
   <zorder>txtAddress</zorder>
   <zorder>btnEditContact</zorder>
   <zorder>btnDeleteContact</zorder>
   <zorder>btnLoadContacts</zorder>
   <zorder>btnSaveContacts</zorder>
   <zorder>btnAddContact</zorder>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
